src/adjoint_sensitivity.jl

```diff
@@ -139,7 +150,8 @@ function AdjointSensitivityIntegrand(sol,adj_sol,alg=SensitivityAlg())
   p = sol.prob.p
   y = similar(sol.prob.u0)
   λ = similar(adj_sol.prob.u0)
-  pf = DiffEqDiffTools.ParamJacobianWrapper(f,tspan[1],copy(y))
+  # we need to alias `y`
```

Comment:
why?

Reply:
Because we need the current `y` to compute Jacobian w.r.t. params, and we never write into `y` inside `ParamJacobianWrapper` so it is fine to alias.